Source/WebKit/WebProcess/GPU/media/MediaPlayerPrivateRemote.cpp

```diff
@@ -894,14 +894,20 @@ NSArray* MediaPlayerPrivateRemote::timedMetadata() const
 String MediaPlayerPrivateRemote::accessLog() const
 {
     auto sendResult = connection().sendSync(Messages::RemoteMediaPlayerProxy::AccessLog(), m_id);
-    auto [log] = sendResult.takeReplyOr(emptyString());
+    auto [log] = sendResult.takeReplyOr(
```

Comment:
I'm not seeing inherent advantage of providing the invokable instead of writing normal linear code for these cases..